contrib/kubernetes/README.md

```diff
@@ -0,0 +1,214 @@
+# Kubernetes managing an etcd2 cluster
+
+The goal of this project is to enable Kubernetes to manage an etcd2 cluster and to keep it healthy
```

Comment:
This project includes guidance and sample files to use Kubernetes for managing and monitoring an etcd cluster. Persistent storage backs each pod as it is created, destroyed, and recreated, in order to provide the reliable state required by etcd. This example constructs an etcd cluster of three nodes, but can be extrapolated to any cluster size.